README.md

```diff
@@ -7,7 +7,7 @@ A super fast, highly extensible markdown parser for PHP
 [![Code Coverage](https://scrutinizer-ci.com/g/cebe/markdown/badges/coverage.png?s=db6af342d55bea649307ef311fbd536abb9bab76)](https://scrutinizer-ci.com/g/cebe/markdown/)
 [![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/cebe/markdown/badges/quality-score.png?s=17448ca4d140429fd687c58ff747baeb6568d528)](https://scrutinizer-ci.com/g/cebe/markdown/)
 
-What is this? <a name="what"></a>
+What is this?
```

Comment:
This might break existing links (i.e. the only available ID there now will be `#what-is-this-` instead of `#what`). How about changing it to `<span id="what"></span>` instead? That validates, and keeps existing links working. (Same for the other ones below.)

Reply:
I really don't care as long as it is valid HTML.  It is currently not valid. You want me to revise?

Reply:
I'm not a maintainer here, so it's not my call, but I'd say it'd be best to maintain backwards compatibility.